src/server/models/users.model.js

```diff
@@ -8,6 +8,7 @@ import mongoose from 'mongoose';
 
 const Schema = mongoose.Schema;
 let userSchema = new Schema({
+  userId: String,
```

Comment:
Oh... so we don't need userId if we need Object id. It will be duplicated. The _id is created automatically and added as _id field...
Did you mean saving userId as _id value???